web/sites/default/config/core.extension.yml

```diff
@@ -12,10 +12,13 @@ module:
   block: 0
   block_content: 0
   block_exclude_pages: 0
+  bootstrap_layout_builder: 0
+  bootstrap_styles: 0
   breakpoint: 0
   bulk_user_registration: 0
   ckeditor: 0
   ckeditor_config: 0
+  clamav: 0
```

Comment:
clamav, pantheon_advanced_cache, and redis lines should be removed from this file. It is part of the local config getting overridden by dev.

Reply:
clamav, pantheon_advanced_cache, and redis removed.

Reply:
Similar to the clamav settings, it is best not to edit these files directly. However, I can't think of a way to cleanly fix this file through configuration changes. You'll need to test the heck out of both your local and multidev to make sure the manual edit doesn't have unintended consequences.